src/openforms/appointments/contrib/jcc/tests/test_plugin.py

```diff
@@ -0,0 +1,144 @@
+import os
+from datetime import date, datetime
+
+from django.test import TestCase
+
+import requests_mock
+from zeep.client import Client
+
+from openforms.appointments.contrib.base import (
+    AppointmentClient,
+    AppointmentLocation,
+    AppointmentProduct,
+)
+
+from ..plugin import Plugin
+
+
+def mock_response(filename):
+    filepath = os.path.abspath(
+        os.path.join(os.path.dirname(__file__), "mock", filename)
+    )
+    with open(filepath, "r") as f:
+        return f.read()
+
+
+class PluginTests(TestCase):
+    maxDiff = 1024
+
+    def setUp(self, *args, **kwargs):
```

Comment:
```suggestion
    def setUp(self):
```
-> always call `super().setUp()`
-> `setUp` doesn't take any args/kwargs